webprotege-gwt-ui-client/src/main/java/edu/stanford/bmir/protege/web/client/perspective/PortletWidgetMapper.java

```diff
@@ -69,19 +74,19 @@ public void setNodePropertiesChangedHandler(Consumer<TerminalNode> nodePropertie
 
     @Override
     public IsWidget getWidget(TerminalNode terminalNode) {
-        GWT.log("[PortletWidgetMapper] Getting widget for TerminalNode: " + terminalNode);
+        logger.info("logger.infoGetting widget for TerminalNode: " + terminalNode);
```

Comment:
I think we should replace the log.info with perhaps log.debug ? leave this kind of logs only for debugging purposes since it generates more traffic through the network.